src/App.js

```diff
@@ -13,6 +16,7 @@ function App() {
   const [productsCategories, setProductsCategories] = useState([]);
   const [featuredBanners, setFeaturedBanners] = useState([]);
   const [featuredProducts, setFeaturedProducts] = useState([]);
+  const [atHomePage, setAtHomePage] = useState(true);
   
   useEffect(() => {
     setProductsCategories(productCategoriesResults);
```

Comment:
Could you clarify where are you using `productsCategories`, `featuredBanners` and `featuredProducts `?

Reply:
I see these imports are used in the `HomePage` component so you can remove them from here.